panther_analysis_tool/main.py

```diff
@@ -1394,6 +1374,7 @@ def setup_parser() -> argparse.ArgumentParser:
     zip_parser.add_argument(path_name, **path_arg)
     zip_parser.add_argument(skip_test_name, **skip_test_arg)
     zip_parser.add_argument(skip_disabled_test_name, **skip_disabled_test_arg)
+    zip_parser.add_argument(available_destination_name, **available_destination_arg)
```

Comment:
Shouldn't this flag also be added to `upload_parser`?

Reply:
Yes, it should, maybe we should add a test for the `upload` sub-command using mocking.

Reply:
Done!